app/components/SearchBar.tsx

```diff
@@ -285,7 +285,7 @@ function buildOptions(rawSearch: string, cluster: Cluster, tokenRegistry: TokenI
             ],
         });
 
-        if (currentEpoch !== undefined && Number(search) <= currentEpoch + 1) {
+        if (currentEpoch !== undefined && Number(search) <= currentEpoch + BigInt(1)) {
```

Comment:
I guess theoretically we could parse the input as a BigInt too, but being careful not to allow fancy integer literal strings that start with `0x`, `0b` and `0o`.

```ts
if (search.length === 0 || /^0\w/.test(search)) return [];

if (currentEpoch !== undefined && BigInt(search) <= currentEpoch + 1n) { /* ... */ }
```

(Also, I'd just use BigInt literals like `1n`)